.vscode/settings.json

```diff
@@ -29,7 +29,7 @@
     "AWS_S3_ECHIMP_BUCKET": "mint-app-echimp-uploads",
     "AWS_ECHIMP_CR_FILE_NAME": "FFS_CR_DATA.parquet",
     "AWS_ECHIMP_TDL_FILE_NAME": "TDL_DATA.parquet",
-    "AWS_ECHIMP_CACHE_TIME_MINS": 180,
+    "AWS_ECHIMP_CACHE_TIME_MINS": "180",
```

Comment:
Not sure if this can just get reverted or not, but seems unrelated here!

Reply:
Thanks Clay, this is needed. I just didn't catch it to previously. Otherwise, VsCode fails to start the debugger.  Surprisingly, it can't parse an int. I was debugging on this feature, so it needs to be here.